src/html/mutation_details_table_view.html

```diff
@@ -30,7 +30,7 @@
 </script>
 
 <script type="text/template" id="mutation_table_protein_change_template">
-	<span class='{{proteinChangeClass}}' alt='{{proteinChangeTip}}'>
+	<span class='simple-tip {{proteinChangeClass}}' alt='{{proteinChangeTip}}'>
```

Comment:
The tool tip for AA change column was intentionally disabled some time ago by removing the `sample-tip` class. I am not sure we want that back at this point. It might be better to discuss it first with the rest of the group before putting it back.